depends/patches/qt/qt.pro

```diff
@@ -0,0 +1,17 @@
+# Create the super cache so modules will add themselves to it.
```

Comment:
Should we move this file and `mac-qmake.conf` to `../configs/qt`?

Reply:
Files in the `depends/patches/<package>/` directory are subjects of a [build hash calculation](https://github.com/bitcoin/bitcoin/blob/25bc840e038f5cc3a1d70d6fbaf1dc2eb2c6e460/depends/funcs.mk#L37-L40). So, I don't think that moving some of them to another directory is a good idea.

Reply:
One could add `depends/configs` to the calculation. No big deal either way, perhaps I'm nitpicking